src/components/app.jsx

```diff
@@ -80,23 +70,49 @@ const App = React.createClass({
                 <div className="col-md-9">
                   <p id="baseline">
                     <span className="message">
-                      <FormattedMessage {...messages.header} />
+                      <FormattedMessage id = "header"/>
+                      <small>
+                        <ExternalLink href = "https://www.openfisca.fr" target = "_blank">
+                          <strong><FormattedMessage id = "learnMore"/></strong>
```

Comment:
It is not the same thing to introduce a specific class and to target already existing markup 🙂
Also, I'm not sure I agree with @fpagnoux on this specific case.